frontend/src/components/validation/bootstrap-form-renderer.js

```diff
@@ -0,0 +1,64 @@
+import {
+    ValidationRenderer,
+    RenderInstruction,
+    ValidateResult
+} from 'aurelia-validation';
+
+export class BootstrapFormRenderer {
+    render(instruction: RenderInstruction) {
+        for (let { result, elements } of instruction.unrender) {
+            for (let element of elements) {
+                this.remove(element, result);
+            }
+        }
+
+        for (let { result, elements } of instruction.render) {
+            for (let element of elements) {
+                this.add(element, result);
+            }
+        }
+    }
+
+    add(element: Element, result: ValidateResult) {
+        if (result.valid) {
+            return;
+        }
+
+        const formGroup = element.closest('.form-group');
+        if (!formGroup) {
+            return;
+        }
+
+        // add the has-error class to the enclosing form-group div
+        formGroup.classList.add('has-error');
+
+        // add help-block
+        const message = document.createElement('span');
+        message.className = 'help-block validation-message';
+        message.textContent = result.message;
+        message.id = `validation-message-${result.id}`;
+        formGroup.appendChild(message);
+    }
+
+    remove(element: Element, result: ValidateResult) {
+        if (result.valid) {
+            return;
+        }
+
+        const formGroup = element.closest('.form-group');
+        if (!formGroup) {
+            return;
+        }
+
+        // remove help-block
+        const message = formGroup.querySelector(`#validation-message-${result.id}`);
+        if (message) {
+            formGroup.removeChild(message);
```

Comment:
either with `{}` or without them

you have above two cases where you use `{}` so to keep the consistency use them here as well